pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -475,6 +523,8 @@ func (f *framework) RunPermitPlugins(
 				klog.Error(msg)
 				return NewStatus(Error, msg)
 			}
+		} else {
+			klog.V(10).Infof("running %q permit plugin for pod %q successfully", pl.Name(), pod.Name)
```

Comment:
easier to just log the status and avoid the `else` block

Reply:
SG